components/Send/Send.tsx

```diff
@@ -575,9 +578,7 @@ const Send: React.FunctionComponent<SendProps> = ({
                         }}
                         value={ta.amount.toString()}
                         onChangeText={(text: string) => updateToField(null, text.substring(0, 20), null, null)}
-                        onEndEditing={(e: any) =>
-                          updateToField(null, Number(e.nativeEvent.text.substring(0, 20)).toFixed(8), null, null)
-                        }
+                        onEndEditing={(e: any) => updateToField(null, e.nativeEvent.text.substring(0, 20), null, null)}
```

Comment:
@zancas The problem here with NaN was: onEndEdiiting had a Number cast which is a problem when the value is not a number, like 0...0 for example... and in this case the value became in NaN. I had to remove Number here.